README.md

```diff
@@ -138,6 +144,15 @@ fi
 
 ![](Screenshots/pre_build_action.png)
 
+Alternatively, if you've installed LicensePlist via CocoaPods the script should look like this:
+
+```sh
+if [ $CONFIGURATION = "Debug" ]; then
+cd $SRCROOT
+${PODS_ROOT}/LicensePlist/portable_licenseplist --output-path $PRODUCT_NAME/Settings.bundle
```

Comment:
I the case of [SwiftLint](https://github.com/realm/SwiftLint) CocoaPods distribution, its executable binary is located at `Pods/SwiftLint/swiftlint`.

So, I think `${PODS_ROOT}/LicensePlist/license-plist --output-path $PRODUCT_NAME/Settings.bundle` is more natural.
(Of course, it is necessary to modify other scripts to fit this command if needed.)